lib/openstudio/workflow/util/energyplus.rb

```diff
@@ -253,35 +253,6 @@ def energyplus_preprocess(idf, logger)
             OpenStudio::Workflow::Util::EnergyPlus.add_energyplus_output_request(idf, object)
           end
 
-          # this is a workaround for issue #1699 -- remove when 1699 is closed.
-          needs_hourly = true
-          needs_timestep = true
-          needs_daily = true
-          needs_monthy = true
-          idf.getObjectsByType('Output:Variable'.to_IddObjectType).each do |object|
-            timestep = object.getString(2, true).get
-            if /Hourly/i =~ timestep
-              needs_hourly = false
-            elsif /Timestep/i =~ timestep
-              needs_timestep = false
-            elsif /Daily/i =~ timestep
-              needs_daily = false
-            elsif /Monthly/i =~ timestep
-              needs_monthy = false
-            end
-          end
-
-          new_objects = []
-          new_objects << 'Output:Variable,*,Zone Air Temperature,Hourly;' if needs_hourly
-          new_objects << 'Output:Variable,*,Site Outdoor Air Wetbulb Temperature,Timestep;' if needs_timestep
-          new_objects << 'Output:Variable,*,Zone Air Relative Humidity,Daily;' if needs_daily
-          new_objects << 'Output:Variable,*,Site Outdoor Air Drybulb Temperature,Monthly;' if needs_monthy
-
-          new_objects.each do |obj|
-            object = OpenStudio::IdfObject.load(obj).get
-            OpenStudio::Workflow::Util::EnergyPlus.add_energyplus_output_request(idf, object)
-          end
-
```

Comment:
It does seem that https://github.com/NREL/OpenStudio/issues/1699 was long fixed by https://github.com/NREL/OpenStudio/pull/1700 indeed.